src/components/Switch/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.css';
+
+class Switch extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            chosenParamIndex: 0,
+        };
+    }
+    handleClick(which) {
+        this.setState({chosenParamIndex: which })
+    }
+
+    getClass(index) {
+        return "Switch__Param " + (this.state.chosenParamIndex === index ? "Switch_activeParam" : "");
```

Comment:
you can use template string here instead

Reply:
fixed